gulpfile.js

```diff
@@ -64,7 +64,13 @@ function contentResources() {
 
 //index and render all readme.md files to HTML
 async function contentPages() {
-  await buildContent(paths.srcContentBase, paths.dist, paths.invaderDefsDir, baseUrl, packageVersion);
+  await buildContent({
+    baseUrl,
+    packageVersion,
+    invaderDefsDir: paths.invaderDefsDir,
+    contentDir: paths.srcContentBase,
+    outputDir: paths.dist
+  });
```

Comment:
In order to reduce the number of function arguments passed around downstream of this function, I found it was clearer to construct a `buildOpts` object here which bundles these things together.